src/blurry_classifier_module.py

```diff
@@ -0,0 +1,210 @@
+            raise ValueError(
+                "Camera name ",
+                {camera_name},
+                "does not match the camera name ",
+                {self.camera_name},
+                "in the config."
+            )
+        im = await self.camera.get_image(mime_type=CameraMimeType.JPEG)
+        classifications = None
+        if return_classifications:
+            classifications = await self.get_classifications(
+                im, 1, extra=extra, timeout=timeout
+            )
+
+        return CaptureAllResult(image=im, classifications=classifications)
+
+    async def get_detections_from_camera(
+        self,
+        camera_name: str,
+        *,
+        extra: Optional[Mapping[str, ValueTypes]] = None,
+        timeout: Optional[float] = None,
+    ) -> List[Detection]:
+        raise NotImplementedError()
+
+    async def get_detections(
+        self,
+        image: ViamImage,
+        *,
+        extra: Optional[Mapping[str, ValueTypes]] = None,
+        timeout: Optional[float] = None,
+    ) -> List[Detection]:
+        raise NotImplementedError()
+
+    async def get_classifications_from_camera(
+        self,
+        camera_name: str,
+        count: int,
+        *,
+        extra: Optional[Mapping[str, ValueTypes]] = None,
+        timeout: Optional[float] = None,
+    ) -> List[Classification]:
+        if camera_name not in (self.camera_name, ""):
+            raise ValueError(
+                "Camera name ",
+                {camera_name},
+                "does not match the camera name ",
+                {self.camera_name},
+                "in the config."
+            )
+        im = await self.camera.get_image(mime_type=CameraMimeType.JPEG)
+        return self.get_classifications(im, 1, extra=extra, timeout=timeout)
+
+    async def get_classifications(
+        self,
+        image: ViamImage,
+        count: int,
+        *,
+        extra: Optional[Mapping[str, ValueTypes]] = None,
+        timeout: Optional[float] = None,
+    ) -> List[Classification]:
+        img = decode_image(image)
+        blurriness = cv.Laplacian(img, cv.CV_64F).var()
+        if blurriness < self.blurry_threshold:
+            return [Classification(class_name="blurry", confidence=0.5)]
+
+        return []
+
+
+    async def get_object_point_clouds(
+        self,
+        camera_name: str,
+        *,
+        extra: Optional[Mapping[str, ValueTypes]] = None,
+        timeout: Optional[float] = None,
+    ) -> List[PointCloudObject]:
+        raise NotImplementedError()
+
+    async def get_properties(
+        self,
+        *,
+        extra: Optional[Mapping[str, ValueTypes]] = None,
+        timeout: Optional[float] = None,
+    ) -> Vision.Properties:
+        return Vision.Properties(
+            classifications_supported=True,
+            detections_supported=False,
+            object_point_clouds_supported=False,
+        )
+
+
+if __name__ == "__main__":
+    asyncio.run(Module.run_from_registry())
```

Comment:
I think these 2 lines should be removed: I don't think there is ever a reason for this file to be the main executable.